tests/bearlib/abstractions/LinterTest.py

```diff
@@ -729,7 +729,7 @@ def test_repr(self):
         self.assertRegex(
             repr(uut),
             '<ManualProcessingTestLinter linter object \\(wrapping ' +
-            re.escape(repr(sys.executable)) + '\\) at 0x[a-fA-F0-9]+>')
+            PathArg(repr(sys.executable)) + '\\) at 0x[a-fA-F0-9]+>')
```

Comment:
Here it's absolutely fine to use `re.escape` since the string is only going to be used as a regex :) So no change needed

Furthermore, this test will fail with `PathArg` on Windows, because the `sys.executable` path in `wrapping 'contains\OS\specific\path\seps'`